src/Servers/Kestrel/Core/src/Internal/Http/HttpProtocol.FeatureCollection.cs

```diff
@@ -250,5 +251,22 @@ void IHttpRequestLifetimeFeature.Abort()
         }
 
         protected abstract void ApplicationAbort();
+
+        Task IHttpResponseStartFeature.StartAsync(CancellationToken cancellationToken)
+        {
+            if (HasResponseStarted)
+            {
+                return Task.CompletedTask;
+            }
+
+            var initializeTask = InitializeResponseAsync(0);
```

Comment:
I'm missing something, why doesn't this just return the initializeTask directly?